tests/test_user_login_and_registration.py

```diff
@@ -52,82 +47,84 @@ def test_user_registration(client):
     (to buy tickets, etc).
     """
     # required for redirects to /
-    create_homepage_in_cms()
```

Comment:
That's cool that this step is no longer necessary - how did you achieve that? I see no changes to `conftest.py` or changes to fixtures.

Reply:
Basically the homepage is just rendered via static view rather than CMS, only content pages are run through CMS.